vaccine_feed_ingest/runners/me/maine_gov/normalize.py

```diff
@@ -24,12 +25,8 @@ def _get_city(site: dict) -> str:
 
 def _get_id(site: dict) -> str:
     name = _get_name(site)
-    city = _get_city(site)
 
-    id = f"{name}_{city}".lower()
-    id = id.replace(" ", "_").replace("\u2019", "_")
-    id = id.replace(".", "_").replace(",", "_").replace("'", "_")
-    id = id.replace("(", "_").replace(")", "_").replace("/", "_")
+    id = location_id_from_name(name)
```

Comment:
To maintain better compatibility with how ids were calculated before, can you pass both name and city to this helper?
```suggestion
    id = location_id_from_name(f"{name}_{city}")
```